cli/cli.go

```diff
@@ -215,6 +220,32 @@ func infoCommand(args []string) {
 	infoTableRender(repository.NewRepository(preferences, name))
 }
 
+
+func renderCommand(args []string) {
+	if len(args) < 1 {
+		terminal.PrintErrorMessage("Error: template must be set")
+		os.Exit(1)
+	}
+
+	template := args[0]
+	output := "index.html"
+
+	if len(args) == 2 {
+		output = args[1]
+	}
+
+	terminal.PrintActionMessage("Rendering template")
+	err := index.ExportIndex(storage.NewStorage(preferences), template, output)
+
+	if err != nil {
+		terminal.PrintActionStatus(1)
+		terminal.PrintErrorMessage("Error: unable to render template")
```

Comment:
This is a very uninformative error message - it doesn't contain any info about what's wrong with the template. You have this info in error struct.